bpfd/src/server/bpf.rs

```diff
@@ -17,19 +17,24 @@ use nix::{
 };
 use uuid::Uuid;
 
-use crate::server::{
-    config::{Config, XdpMode},
-    errors::BpfdError,
+use crate::{
+    proto::bpfd_api::ProceedOn,
+    server::{
+        config::{Config, XdpMode},
+        errors::BpfdError,
+    },
 };
 
-const DEFAULT_ACTIONS_MAP: u32 = 1 << 2;
+const DEFAULT_ACTIONS_MAP: u32 =
```

Comment:
When I refactored this last I was really careful to try and create some meaningful boundaries between the different modules in this crate.

It would be great if there were a way not to require `proto::bpfd_api::ProceedOn` here.
Maybe we could add the defaults in the `rpc` module?

Reply:
Still need to work on this one. Just hard-coded it to 2 and 31 for now.